app/src/main/java/com/google/samples/apps/sunflower/compose/plantdetail/PlantDetailView.kt

```diff
@@ -227,7 +228,7 @@ fun PlantDetails(
                 maxOf(candidateHeight, 1.dp)
             },
             onFabClick = callbacks.onFabClick,
-            onGalleryClick = callbacks.onGalleryClick,
+            onGalleryClick = { callbacks.onGalleryClick(plant) },
```

Comment:
Out of scope for this PR but we should eliminate the `callbacks` at some point

Reply:
Agreed